db/seeds.rb

```diff
@@ -1,6 +1,6 @@
 
-user = User.where(email: 'jorge@testing.com').first_or_create(password: 'jorgeeats', password_confirmation: 'jorgeeats')
-user2 = User.create( email: 'george@testing.com',password: 'georgeeats', password_confirmation: 'georgeeats')
+user1 = User.where(email: 'jorge@testing.com').first_or_create(password: 'jorgeeats', password_confirmation: 'jorgeeats')
+user2 = User.where( email: 'george@testing.com').create(password: 'georgeeats', password_confirmation: 'georgeeats')
```

Comment:
I would swap out `.create` for `.first_or_create` on line 3 so that you don't run into issues if you have to reseed your database.